src/exprs.c

```diff
@@ -911,12 +911,12 @@ Obj             EvalPermExpr (
             ptr2[k-1] = ptr4[k-1];
         };
         RetypeBag( perm, T_PERM2 );
-        ResizeBag( perm, m * sizeof(UInt2) );
+        ResizeBag( perm, SIZEBAG_PERM2(m));
```

Comment:
The first commit in this PR has message "Use new SIZEBAG_PERM<n> functions rather than making assumptions." -- but actually, those new functions are only added by a later commit, meaning this commit is uncompilable.